contrib/kubernetes/datahub/charts/datahub-mae-consumer/templates/deployment.yaml

```diff
@@ -33,45 +33,24 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           env:
             - name: KAFKA_BOOTSTRAP_SERVER
-              valueFrom:
```

Comment:
I understand the underlying advantage of using configmap but it also adds on to security risk wherein we are not aware of changes made to configmap resources. Also, for higher envs like prod and other client-facing envs, I(CD) want to be aware of such changes and hence can version the deployment based on changes so that in future if these new changes in configmap is responsible for any failure, we can directly roll-back.

Reply:
I'm not sure I understand how configmaps are less secure. If others don't have an issue with baking in environmental variables in the deployment, it's not a deal breaker for me.